apps/bfd-server/bfd-server-launcher/pom.xml

```diff
@@ -121,6 +121,23 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<!-- API, java.xml.bind module -->
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.2</version>
+		</dependency>
+		<!-- Runtime, com.sun.xml.bind module -->
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+			<version>2.3.1</version>
+		</dependency>
+		<dependency>
+		<groupId>javax.servlet</groupId>
+    <artifactId>javax.servlet-api</artifactId>
+    <version>4.0.1</version>
+	</dependency>
```

Comment:
formatting and dependency management (would also include the servlet api in that lis)